address-space-controller/pom.xml

```diff
@@ -44,6 +44,11 @@
       <artifactId>vertx-proton</artifactId>
       <scope>compile</scope>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web</artifactId>
+      <scope>compile</scope>
+    </dependency>
```

Comment:
I would prefer og we didnt introduce another dependency here, but used the existing vertx http server